drivers/serial/Kconfig.ns16550

```diff
@@ -34,6 +34,11 @@ config UART_NS16750
 	  This enables support for 64-bytes FIFO and automatic hardware
 	  flow control if UART controller is 16750.
 
+config UART_NS16750_EXTFIFO
+	bool "Enable UART features (128-bytes FIFO and auto flow control)"
```

Comment:
Do not start Kconfig names with "Enable" since that is implied; something like "Extended UART 16750 (128-bytes FIFO and auto flow control)" should do.

Also this needs to be a `choice` since `UART_NS16750` and `UART_NS16750_EXTFIFO` are mutually exclusive.

Reply:
what do you mean by "choice"?

Reply:
Also now that I think about it, if it has 128-byte FIFO, then it is basically 16950, isn't it?

Reply:
I think that's the case... should I change UART_NS16750_EXTFIFO to UART_NS16950?

Reply:
> should I change UART_NS16750_EXTFIFO to UART_NS16950?

Yes, that would make more sense, IMO.

Reply:
this has been changed